styles.css

```diff
@@ -361,7 +358,7 @@
 }
 
 .project-container {
-  background: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%), url("././images/project-setup3.jpg");
+  background: linear-gradient(180.45deg, rgba(38, 38, 38, 0.9) 0.75%, rgba(38, 38, 38, 0.9) 61.94%), url("././images/project-setup3.jpg");
```

Comment:
- [ ] I understand to use an image as a background could be helpful, at the same time, use it on `<img>` inside HTML code is part of this project to understand how images work with this important tag. I invite you to, some of the images you use on the project, include them on HTML code with the `<img>` tag. I'm pretty sure you'll get another way to work with images on this way. 🐶